### PR TITLE
Standardize code formatting

### PR DESCRIPTION
**Pull request by sashadev-sky**
Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [ ] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

formatting: 
- [ ] 2 space indents
- [ ] single quotes

### CODE REVIEW

**Comment by sashadev-sky** [comment]
@rexagod im going to standardize the code formatting so that all src files use 2 space indent, and all test files use 4 space indent, because right now it is not standard for every file. 
Do you like this idea? I have only done 1 file so far but if you approve i'll update the rest

**Comment by rexagod** [comment]
Sounds good! And what are your thoughts on using a linter to enforce such style in the future? We can add `--fix` flag at runtime to auto-format all files to this particular style at build.

Or better yet, use husky to add pre-commit hooks that check linting? That should automate the formatting stuff for us, I guess.

**Comment by sashadev-sky** [comment]
@rexagod Ok that sounds like a much better idea than this PR do you want to open one with Husky?

**Comment by rexagod** [comment]
On it.

**Comment by rexagod** [comment]
Okay just a question before I push on this, do you want to apply linting _only_ for the spacing stuff, or do you think it'd be better to apply an [already existing linting style](https://github.com/dustinspecker/awesome-eslint#configs) to this lib?

**Comment by sashadev-sky** [comment]
@rexagod oh really good idea!! I tend to look up google's style guide for best practice, but if you have experience with using a particular one and prefer it let's use that! Also don't know the diff but in the past Airbnb style guide was recommended to me.

This is actually an important maintenance move that i've been thinking we should do for a while so glad we're getting to it

**Comment by rexagod** [comment]
@sashadev-sky `--fix` fixed most of the errors while 441 of them required manual attention. So I've fixed them up now, and added some of them to warnings for later.

x

Also, I would recommend you to install the eslint extension so that you can easily see the errors/warnings as you type.

lint

**Comment by rexagod** [comment]
Actually let me just fix those warnings too, if 400, why not 23 more? 🙃

**Comment by rexagod** [comment]
Done!

**Comment by rexagod** [comment]
cc @sashadev-sky @jywarren

**Comment by sashadev-sky** [comment]
@rexagod awesme im gonna pull this and check it out. i guess i will rebase it to because that was all me.. 😫

**Comment by sashadev-sky** [comment]
@rexagod wait how are you using `let` and `const` everywhere? isnt that only for ES6

**Comment by sashadev-sky** [comment]
@rexagod tests are not passing. 

image

I believe this is due to your usage of `let` and `const`. I have tripped over this to a few times in this lib we can only use `var` until make ES6 compatible.

**Comment by rexagod** [comment]
@sashadev-sky Check now.

**Comment by sashadev-sky** [comment]
@rexagod ok awesome! Thank you!!! I am going to rebase and merge this an open a follow-up about some questions I have

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules
+test/src/util
+test/src/*Spec.js
+Gruntfile.js
```
Comment (sashadev-sky) [inline .eslintignore]: Why do we want to be ignoring `Gruntfile.js`?

**Comment by sashadev-sky** [comment]
actually @rexagod I left a review comment that I would like to address quickly before merge

File: .eslintignore (hunk 1 above)
Comment (sashadev-sky) [reply .eslintignore]: Ohh its .eslintignore gotchya

**Comment by sashadev-sky** [comment]
Ok this is ready! Disregard last comment. I'll open the follow up :)

**Pull request merged**